### ISSUE TITLE
Re-release dirsearch version 0.4.1

### ISSUE THREAD

**Issue by shelld3v** [maurosoria/dirsearch#661]
### What is the current behavior?

If you go to the home page of dirsearch, on the left, you will see the latest version is dirsearch v0.4.0, which is wrong. Can you please delete the v0.4.1 tag then release it again, so we can fix this problem?

### What is the expected behavior?

It should show the latest version is dirsearch v0.4.1

### Any additional information?

```
Releases 15
    dirsearch v0.4.0 (Latest)
    on Sep 27
+ 14 releases
```

**Comment by shelld3v** [comment]
Also merge my bug fixes before release it again

**Comment by shelld3v** [comment]
The fact that we are keep generating new bugs after code updates, and I hate to see our stargazers use our buggy version. There must be some way to reduce this! Maybe a tester? For now, I will stop updating core functions (still, but less), because I don't want to make more problems for our users (and our users will choose another tool T_T).

**Comment by maurosoria** [comment]
Hello @shelld3v ,

What you are saying is the result that we didn't wait the proper time to check everything is working properly before drafting.

But, I think it's great, sometimes is necessary to happen, and many folks are coming around to report it, so you should feel great.

I tell you this, let's fix everything and I'll draft v4.0.2 in a short time (couple of days). Please, do not add other functionality and just be a little patient.


Thanks as always @shelld3v 👍 

Regards,
Mauro
